blog/2024-09-24-python-313.md

```diff
@@ -0,0 +1,65 @@
+---
+authors:
+  - isuruf
+tags: [infrastructure]
+---
+
+# Python 3.13 Release candidate builds on conda-forge
+
+conda-forge now supports Python 3.13 release candidates on conda.
+You can create a new environment with Python 3.13 by running the
+command:
+
+    conda create -n py313 python=3.13 -c conda-forge/label/python_rc -c conda-forge
+
+<!-- truncate -->
+
+This will create a new environment with Python 3.13 with the global
+interpreter lock (GIL) enabled. A migration is underway that builds
+Python extensions like those included in `numpy` and `scipy` as conda packages. The migration
+is 50% complete at the time of writing. Once the first final release of
```

Comment:
Maybe add a link to the status page so folks can see an update on this?

Reply:
```suggestion
is [50% complete](/status/migration/?name=python313) at the time of writing. Once the first final release of
```